wpeverest/user-registration
# UR-980 Fix - Form design conflict with themes using bootstrap

### All Submissions:






### Changes proposed in this Pull Request:
Install the attached theme and check the design of UR forms and the Edit Profile form in the My Account section. This PR fixes this design conflict.
[shelly-v.1.1.0.zip](https://github.com/wpeverest/user-registration/files/11878415/shelly-v.1.1.0.zip)


### How to test the changes in this Pull Request:

1. Check to see if the design is fixed in the theme provided above, also verify if the design is not distorted in default and popular themes.

### Types of changes:




### Other information:




### Changelog entry

> Enter a short summary of all changes on this Pull Request. This will appear in the changelog if accepted.
